CONTRIBUTING.md

```diff
@@ -301,3 +305,28 @@ Then in the `pybricks-micropython` directory:
     yarn build:debug
 
 [wasm]: https://webassembly.org/
+
+Build and deploy firmware
+-------------------------
+
+### Build the firmware
+
+Pick your Hub from the `bricks` sub-directory you want to compile.
+
+    poetry shell
+    make mpy-cross -j8
+    make -C bricks/primehub -j8
+       
+### Deploy the firmware to a hub
+
+First, follow the guide to turn your Hub into DFU mode:
+1. https://pybricks.com/install, or 
+2. https://dfu.pybricks.com for Spike Prime or Mindstorms Inventor
+                
+Execute the deployment:
+
+    make -C bricks/primehub -j8 deploy-dfu
+
+For Inventor Hub, use the following command:
```

Comment:
This should not be needed for the reason above.

Reply:
Done.